website/content/docs/secrets/pki/considerations.mdx

```diff
@@ -616,6 +617,31 @@ correctly throughout all parts of the PKI. In particular, CRLs embed a
 track revocation, but note that performance characteristics are different
 between OCSP and CRLs.
 
+## Issuer Storage Migration Issues
+
+When Vault migrates to the new multi-issuer storage layout on releases prior
+to 1.11.6, 1.12.2, and 1.13.0, and storage write errors occur during the mount
```

Comment:
Drop 1.13.0 from here I'd say? 

I'd also mention that multiple issuers/parents are needed within the mount for the ca_chain to be miscalculated?

Reply:
I dunno, wouldn't it be good to mention that 1.13.0 won't have this problem preemptively, as the other versions are minor patch releases post-GA? My 2c., happy to delete though.

Ah, I figured that'd be implied by the fact its only the `ca_chain` that's wrong, but I can make that more clear.

Reply:
I've dropped the `.0` from 1.13 in this update.